src/pages/gateway/release-notes.md

```diff
@@ -18,31 +18,31 @@ The following sections indicate when updates were made to API Mesh for Adobe Dev
 
 This release contains the following changes to API Mesh:
 
-### Bug fixes
+### Bugfixes
```

Comment:
Here MS recommends avoiding the term entirely: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/b/bug-fix

Reply:
Acrolinx allows the term _Bugfix_, but suggests _Bug fixes_ for plural

Reply:
Reverted to _Bug fixes_.